apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/AbstractClassEnhancePluginDefine.java

```diff
@@ -70,6 +75,25 @@
                 }
             }
         }
+        List<WitnessMethod> witnessMethods = witnessMethods();
```

Comment:
This existing logic should move into the `WitnessClassFinder` from my understanding. And you should add UTs for WitnessClassFinder.

Reply:
I think The WitnessClassFinder class name should be changed If the logic is moved.
What's the meaning of UTs here?

Reply:
`WitnessClassFinder` -> `WitnessFinder` should be enough.
UT means Unit Test, to verify the method you added.